### ISSUE TITLE
Multi-day events

### ISSUE THREAD

**Issue by zenhack** [zenhack/sandcal#25]
I would like to be able to enter an event into my calendar that spans more than one day, e.g. a camping trip.

**Comment by ocdtrekkie** [comment]
I think this is not really a separate issue, so I'll just note here: I tried to put an event in that started at 11 PM and ended at 12 AM, and the weekly view was VERY confused, which makes sense, considering I must've technically submitted an event that ended before it began.

**Comment by zenhack** [comment]
Yeah, I'm not surprised the weekly view did something weird with that. Probably we should do some validation to keep you from doing that in the first place.
